data/input_files/tests/thermal_conduction/static_reaction_exact.lua

```diff
@@ -0,0 +1,66 @@
+-- Comparison information
```

Comment:
This was verified to be correct by taking the L2 error against the exact solution in a C++ driver code.